packages/patternfly-4/react-core/src/components/DataList/DataListItemCells.js

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+import { css } from '@patternfly/react-styles';
+import PropTypes from 'prop-types';
+import styles from '@patternfly/patternfly/components/DataList/data-list.css';
+
+const DataListItemCells = ({ className, dataListCells, ...props }) => (
+  <div className={css(styles.dataListItemContent, className)} {...props}>
+    {dataListCells}
+  </div>
+);
+
+DataListItemCells.propTypes = {
+  /** Additional classes added to the DataList item Content Wrapper.  Children should be one ore more <DataListCell> nodes */
+  className: PropTypes.string,
+  /** Array of <DataListCell> nodes that are rendered one after the other. */
+  dataListCells: PropTypes.arrayOf(PropTypes.node).isRequired,
```

Comment:
We already compose
```
<DataList>
  <DataListItem>
      <DataListItemCells />
  </DataListItem>
</DataList>
```
Then why not also `<DataListCell>` into DataListItemCells?

Reply:
I want to enforce the consumer using DataListItemsCell wrapper around the DataListItems.  That is why I have the prop.  Does that answer you question?  I am not sure I understand it.

Reply:
Consumer could still pass anything they want into the dataListCells prop... why not
```
<DataListItemCells>
  <DataListCell>...</DataListCell>
</DataListItemCells>
```